OrbitBase/Logging.cpp

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 #include <OrbitBase/Logging.h>
```

Comment:
Just mentioning that ideally these would be changed to using quotes.

Reply:
Yes. I can easily influence that for new includes. But as far as I can see iwyu does not support that for existing and working includes. I guess we have to find another way to fix that.

Reply:
Yep, I know (I remember we talked about this), I guess we can just do it manually eventually, there shouldn't be many left.